tests/mock_cloud_sdk.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+from pasqal_cloud.job import CreateJob
+from pasqal_cloud.device import BaseConfig, EmulatorType
+from pasqal_cloud.batch import Batch
+from pasqal_cloud.utils.responses import PaginatedResponse
+
+from uuid import uuid4
+
+from unittest.mock import MagicMock
+from typing import Any
+
+
+class MockSDK:
+    """Helper class to mock the cloud SDK and skip the API calls.
+
+    Warning: This implements only a small subset of the functions available
+        in the cloud SDK; focusing on the functions used by qek.
+        This should be extended to support all methods and moved to the
+        pasqal-cloud repository so that it can be reused for all future
+        libraries using the SDK.
+
+    TODO:
+      - mock execution of jobs and batches (pending => running => done)
+      - set proper results for jobs (right now its entirely mocked)
+    """
+
+    def __init__(self) -> None:
+        self.batches: dict[str, Batch] = {}
+
+    def get_device_specs_dict(self) -> Any:
+        with open("tests/fixtures/device_specs.json", "r") as f:
```

Comment:
Nit: Doc.

Reply:
I also think this should be provided by the `pasqal_cloud` package.

Reply:
Added some documentation